utility/types/proto/message.proto

```diff
@@ -4,19 +4,13 @@ package utility;
 option go_package = "github.com/pokt-network/pocket/utility/types";
 
 import "google/protobuf/any.proto";
-
-enum UtilActorType { // TODO (team) deprecate for persistence/actorType
-  App = 0;
-  Node = 1;
-  Fish = 2;
-  Val = 3;
-}
+import "actor.proto";
 
 message MessageSend {
   bytes from_address = 1;
   bytes to_address = 2;
   string amount = 3;
-  UtilActorType actor_type = 4;
```

Comment:
It's unclear to me what `actor_type` implies.

Is it the type of the actor that created the message? If so, I think we should rename it to just `actor` or `author_type` or `sender_actor_type`.

Open to alternatives, but I just want the name to be self documenting and clear.